frontend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "frontend",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

Comment:
Maybe, we can define dev mode for the plugin. So, we don't need to use build:jenkins for checking the plugin UI result when developing the UI.